message.go

```diff
@@ -80,6 +80,9 @@ type Message struct {
 	// A list of reactions to the message.
 	Reactions []*MessageReactions `json:"reactions"`
 
+	// Whether the message is pinned or not.
+	Pinned bool `json:"pinned"`
+	
```

Comment:
The empty line there should not have spaces

Reply:
Ah thanks will edit that